arcade-services.sln

```diff
@@ -88,7 +88,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DependencyUpdater.Tests", "
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Validation", "Validation", "{F92E1B5C-26D4-4244-A4F9-41E22BA8E1CE}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.DotNet.Services.Utility", "src\Shared\Microsoft.DotNet.Services.Utility\Microsoft.DotNet.Services.Utility.csproj", "{4CDCFCAA-4B1E-47B7-B6F0-81FFA69BD131}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Services.Utility", "src\Shared\Microsoft.DotNet.Services.Utility\Microsoft.DotNet.Services.Utility.csproj", "{4CDCFCAA-4B1E-47B7-B6F0-81FFA69BD131}"
```

Comment:
I wish I knew what this guid was, and why everyone's PR seems to flop it. :-(

Reply:
same... any way we could learn that? also should I flop it back or just let it change?